go/runtime/registry/registry.go

```diff
@@ -236,6 +256,34 @@ func (r *runtimeRegistry) Cleanup() {
 	}
 }
 
+func (r *runtimeRegistry) FinishInitialization(ctx context.Context) (rerr error) {
+	r.RLock()
+	defer r.RUnlock()
+
+	for _, rt := range r.runtimes {
+		if rt.storage == nil {
```

Comment:
```suggestion
		if rt.storage != nil {
			continue
		}
```
And outdent the block below.